### PR TITLE
Wrong parseInt() short hand comment fixed.

### LINKED ISSUE #475: Wrong comment "leading `+` is short for `parseInt()`"
https://github.com/vitaly-t/pg-promise/blob/d261e894e76744e522266ae01ab2f1c1785a260d/lib/database.js#L1152

### Wrong comment found in documentation.
**Wrong ->** leading `+` is short for `parseInt()`
**Correct ->** leading `+` is short for `parseFloat()`

Correct short hand for parseInt() is trailing `|0` 
To see this working, try `console.log(+"1.6")` and `console.log("1.6"|0)` in Console

### PR DESCRIPTION
**Pull request by nidin**
Closes #475

### CODE REVIEW

File: lib/database.js [hunk 1]
```diff
@@ -1149,7 +1149,7 @@ function Database(cn, dc, config) {
          * @example
          *
          * db.each('SELECT id, code, name FROM Events', [], row => {
-         *     row.code = +row.code; // leading `+` is short for `parseInt()`
+         *     row.code = row.code|0; // trailing `|0` is short for `parseInt()`
```
Comment (vitaly-t) [inline lib/database.js]: I understand that `| 0` is indeed the shorter version, but in this case I'd rather use `parseInt` in documentation, not to confuse developers with some strange-looking syntax :smile:

**Comment by vitaly-t** [comment]
I have committed the change to use `parseInt` instead. Thank you for the PR!

File: lib/database.js (hunk 1 above)
Comment (mkmpvtltd1) [reply lib/database.js]: it's not strange-lloking syntaxt :rofl:

**Comment by nidin** [comment]
@vitaly-t No problem, the problem was the wrong information in documentation. It can happen that some new JS devs read your doc and use `+` to parse int that can lead to serious problems.  JavaScript engines convert underlaying data type to float 64 when you use leading `+` and to int 32 when you use trailing `|0` <- this is basically a bitwise operation which can only possible with integers. therefore prior to this operation engine will convert data to int 32. However, I have to correct that `parseInt()` and `|0` is not 1:1 equal for example `console.log("1abc" |0)` and `console.log(parseInt("1abc")` same thing will happen with `parserFloat("1.25abc")` and `+"1.25abc"`. so in anyway `+` != `parseFloat()` and `|0` != `parseInt()`
Thanks for your quick fix.

**Pull request closed without merging**